src/eko/thresholds.py

```diff
@@ -260,3 +260,29 @@ def nf(self, q2):
         """
         ref_idx = np.digitize(q2, self.area_walls)
         return 2 + ref_idx
+
+
+def is_downward_path(path):
+    """
+    Determine if a path is downward:
+        - in the number of active flavors when the path list contains more than one `PathSegment`,
+           note this can be different from each `PathSegment.is_downward`
+        - in :math:`Q^2` when just one single `PathSegment` is given
+
+    Parameters
+    ----------
+        path: list(`eko.thresholds.PathSegment`)
+
+    Returns
+    -------
+        is_downward: bool
+            True is number of flavor is decreasing
+        shift: 3, 4
+            shift to number of light flavors
+    """
+    if len(path) == 1:
+        is_downward = path[0].is_downward_q2
+    else:
+        is_downward = path[1].nf < path[0].nf
+    shift = 4 if is_downward else 3
```

Comment:
I see it was just convenient to return them together...

Reply:
Not even always, since sometimes you trash the second one:

https://github.com/N3PDF/eko/blob/a0b5017aacdf7ba0b23f420a1e50e25fe8b30549/src/eko/evolution_operator/grid.py#L246

So just return the first, the second one you obtain by a trivial function applied on the first returned value:
```py
def flavor_shift(is_downward):
    return 3 if not is_downward else 4
```